test/functional/normalization/multiline_exception_vector_test.go

```diff
@@ -0,0 +1,186 @@
+//go:build vector
```

Comment:
It should be possible to modify and add to the existing test to support vector: https://github.com/openshift/cluster-logging-operator/blob/master/test/functional/normalization/multiline_exception_test.go#L1

Suggestions:
* Modify to use Elasticsearch as the output
* Remove tag hints at the op of the file

I'm otherwise impressed that vector is capable of supporting our needs without writing our own rust code

cc @alanconway

Reply:
> It should be possible to modify and add to the existing test to support vector: [`master`/test/functional/normalization/multiline_exception_test.go#L1](https://github.com/openshift/cluster-logging-operator/blob/master/test/functional/normalization/multiline_exception_test.go?rgh-link-date=2022-11-15T19%3A03%3A31Z#L1)

I tried that but not work for me, got some problem with images. I will take a look one more time 

> 
> Suggestions:
> 
> * Modify to use Elasticsearch as the output
Maybe it was the problem because it uses fluentd as the output 

> * Remove tag hints at the op of the file
> 
> I'm otherwise impressed that vector is capable of supporting our needs without writing our own rust code
> 
> cc @alanconway

Reply:
@vparfonov you will need to modify the test to read and write to elasticsearch since that is the only common output

Reply:
> @vparfonov you will need to modify the test to read and write to elasticsearch since that is the only common output

sorry, but i don't follow you

Reply:
We should be able to use the existing test for both vector and fluentd.  It requires, however, to change the output type to Elasticsearch since this is the only one which is common to both which we can control in the functional framework